sky/global_user_state.py

```diff
@@ -255,12 +326,75 @@ def set_cluster_metadata(cluster_name: str, metadata: Dict[str, Any]) -> None:
         raise ValueError(f'Cluster {cluster_name} not found.')
 
 
+def get_cluster_usage_intervals(cluster_hash: str) -> Optional[Dict[str, Any]]:
```

Comment:
Add prefix `_` to the function name for the private functions in this file: `get_cluster_usage_intervals`, `set_cluster_usage_intervals`, `get_cluster_hash`.